scripts/create_sample_worksheet.py

```diff
@@ -67,11 +67,11 @@ def create(self):
         ).split('\n')
         if re.match(SampleWorksheet._FULL_UUID_REGEX, worksheet_uuids[0]):
             print(
-                'There is already an existing {} with UUID {}. Skipping creating a sample worksheet...'.format(
+                'There is already an existing {} with UUID {}. Removing and creating a new sample worksheet...'.format(
                     self._worksheet_name, worksheet_uuids[0]
                 )
             )
-            return
+            run_command([self._cl, 'wrm', '--force', self._worksheet_name])
```

Comment:
This logic was added so the same worksheet (including both small and large worksheet) does not accidentally get recreated multiple times, since it can even take some time to even create a small worksheet. I think a better fix here would be to refactor the logic, so we can skip creating a new worksheet if one exists, but still have the ability to validate its output.

Reply:
one thing we could do is to save the expected lines somewhere

Reply:
It seems like the build is failing now. 

I still think it will be cleaner if we refactor so that we don't need these extra files. We can just build the expected lines irregardless of whether or not a  worksheet was created.

Reply:
That doesn't seem feasible: for valid bundles and valid worksheets, I guess you can hard code the expected lines to be self._entities_count, but what about search ",png", "tsv" "gif" bundles? I'm not sure where to find the exact numbers of expected lines to add for these results without actually running the search command